account.go

```diff
@@ -33,41 +23,33 @@ type account struct {
 type Account interface {
 	Client
 	Address() (btcutil.Address, error)
-	SerializedPublicKey() []byte
+	SerializedPublicKey() ([]byte, error)
 	SendTransaction(
+		ctx context.Context,
 		script []byte,
 		fee int64,
+		preCond func(*wire.MsgTx) bool,
 		f func(*txscript.ScriptBuilder),
-		preCon, postCon func(*wire.MsgTx) bool,
+		postCon func(*wire.MsgTx) bool,
```

Comment:
One is called `preCond` and the other is called `preCon`?

Reply:
done